sls-public-demo.html

```diff
@@ -105,6 +105,14 @@
 
 </div>
 
+<p>If an image in the PUBLIC group has a file attachment, this can be downloaded using a link with the syntax:</p>
+    
+<p class="code-example">"https://nightshade.openmicroscopy.org/webclient/annotation/358508/"</p>
+    
+<p>For example:</p>
+    
+<p>Download the P-TRE_19_R3D_D3D results as an spreadsheet <a href="https://nightshade.openmicroscopy.org/webclient/annotation/358508/">intensity results</a></p>
```

Comment:
an -> a
Also, reads a bit strangely, should it be "Download… as a spreadsheet **of** intensity results"?